docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -688,7 +773,11 @@ public class CustomTenantConfigResolver implements TenantConfigResolver {
 
 The `OidcTenantConfig` returned from this method is the same used to parse the `oidc` namespace configuration from the `application.properties`. You can populate it using any of the settings supported by the `quarkus-oidc` extension.
 
-== Tenant Resolution for OIDC 'web-app' applications
+=== Tenant Resolution for OIDC `web-app` applications
+
+Simplest option for resolving OIDC `web-app` application configiration is to follow steps described in the <<default-tenant-resolver, Default Resolution>> section.
```

Comment:
```suggestion
The simplest option for resolving OIDC `web-app` application configuration is to follow the steps described in the <<default-tenant-resolver>> section.
```
Anchor change bit not essential, but just flagging that Guillaume wrote a nice script as part of the doc build to convert Naked anchors into an xref that contains the section title text.

Reply:
@michelle-purcell, George fixed the `configiration` typo with his suggestion earlier, so I could not apply your suggestion directly, thanks